Formula/xmake.rb

```diff
@@ -1,7 +1,7 @@
 class Xmake < Formula
   desc "A cross-platform build utility based on Lua"
   homepage "https://xmake.io/"
-  url "https://github.com/xmake-io/xmake/releases/download/v2.2.9/xmake-v2.2.9.tar.gz"
+  url "https://cdn.jsdelivr.net/gh/xmake-mirror/xmake-releases@2.2.9/xmake-v2.2.9.tar.gz"
```

Comment:
Please use the original upstream instead of a mirror

Reply:
ok, I will restore it. 

But it is difficult to access github and bintray fastly in China. Are there any other good other good solutions?

Reply:
I have fixed url.